qln/netio.go

```diff
@@ -179,16 +179,19 @@ func (nd *LitNode) OutMessager() {
 type PeerInfo struct {
 	PeerNumber uint32
 	RemoteHost string
+	LitAdr 	   string
 	Nickname   string
 }
 
 func (nd *LitNode) GetConnectedPeerList() []PeerInfo {
 	var peers []PeerInfo
 	for k, v := range nd.RemoteCons {
 		var newPeer PeerInfo
+		var pubArr,_ = nd.GetPubHostFromPeerIdx(k)
```

Comment:
Since GetPubHostFromPeerIdx reads data from the database, it seems to make sense to swap this out for:

```go
var pubArr [33]byte
copy(pubArr[:], v.Con.RemotePub.SerializeCompressed())
```
Since the remote pubkey is already in memory.